src/main/java/edu/kis/powp/jobs2d/command/ComplexCommand.java

```diff
@@ -20,7 +20,7 @@ public ComplexCommand(List<DriverCommand> commandList) {
 
 	public void appendCommand(DriverCommand command) {
 		this.commands.add(command);
-	}
+  }
```

Comment:
You have ignored previous comments: e.g. "remove appendCommand. We have builder now."
Revise previous reviews and apply all suggestions.